build/tutorials/tutorials-contest/NFT-Marketplace-on-Avalanche/README.md

```diff
@@ -0,0 +1,1828 @@
+- Basic knowledge of [Solidity](https://docs.soliditylang.org/en/v0.8.6/#) language.
+  - We will be writing our smart contracts in **solidity**. A piece of basic knowledge is required for you to understand the concepts fully.
+
+## Requirements
+
+- [Node JS](https://nodejs.org/en) and [npm](https://www.npmjs.com/) must be installed.
+- [Hardhat](https://hardhat.org/) must be installed.
+- [Metamask](https://metamask.io) extension must be installed on your browser.
+- [create-react-app](https://www.npmjs.com/package/create-react-app) must be installed.
+
+# Getting Started
+
+## Building the workspace
+
+Let's get started by setting up our workspace using [Hardhat](https://hardhat.org/).
+
+- `npx hardhat init` -> choose sample project
+- Delete the `Greeter.sol` file inside contracts
+
+This will set up our initial workspace.
+
+## Creating the NFT Token
+
+First, we need to create an NFT token that will be displayed in our marketplace.
+
+Create a simple ERC-721 token.
+
+[NFT.sol](contracts/NFT.sol)
+
+```solidity
+contract NFT is ERC721 {
+  using Counters for Counters.Counter;
+  Counters.Counter private _tokenIds;
+  mapping (uint => uint) public itemValue;
+  uint maxValue = 10000; // Max value of an item
+
+  constructor() ERC721("Super NFT", "SPRNFT") {}
+
+  /**
+    Returns a random number
+  */
+  function random() private view returns (uint) {
+        return uint(keccak256(abi.encodePacked(block.difficulty, block.timestamp, block.number)));
+  }
+
+  function myItems() external view returns (uint[] memory items) {
+    // Returns an array of items that the user owns
+    items = new uint[](balanceOf(msg.sender));
+    uint _counter = 0;
+    for(uint i = 1; i < _tokenIds.current(); i++) { // i = 1 because the token counter is increased before the id is assigned
+      if(ownerOf(i) == msg.sender) { // if the user owns the item
+        items[_counter] = i; // add the item to the array
+        _counter++; // increase the counter
+      }
+    }
+    return items;
+  }
+
+  function getItem() payable public returns (uint256) {
+    require(msg.value == 0.5 ether); // 0.5 AVAX is the cost of an item
+    _tokenIds.increment(); // Increase the counter
+    uint256 newItemId = _tokenIds.current(); // Get the current counter value
+    _mint(msg.sender, newItemId); // Mint the new item to the user
+    itemValue[newItemId] = random() % maxValue; // Set the item value to a random number modulus used to make sure that the value isn't bigger than maxValue
+    return newItemId; // Return the new item id
+  }
+}
+```
+
+> Note: Secure random-number generation in the blockchain is a very difficult problem. Our method here is insecure, but since this is just an example, it will be good enough for our purposes.
```

Comment:
Can you make this line bold and/or add `ALERT` in all caps below and above it? We want to be sure that devs are clear this isn't a best practice.